### PR TITLE
Feature/admin user forms

### PR DESCRIPTION
**Pull request by cooper-jackson**
# Description

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [x] `npm test`

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: src/components/AdminDashboard/AntModalComponent/AntModal.js [hunk 1]
```diff
@@ -1,55 +1,52 @@
...
+    //   .post(
+    //     'http://hrf-bw-labs37-dev.eba-hz3uh94j.us-east-1.elasticbeanstalk.com/form-out',
+    //     sendObj
+    //   )
+    //   .then(res => {
+    //     console.log(res);
+    //   })
+    //   .catch(err => {
+    //     console.log(err);
+    //   });
```
Comment (alvin-joseph) [inline src/components/AdminDashboard/AntModalComponent/AntModal.js]: Everything looks awesome! Nice work guys! I was wondering about this section though. Is the commented out part for the next Labs group? Also there is a console.log on line 29

**Review by alvin-joseph — APPROVED** [review]
Wow you guys went above and beyond! Nice work! It all looks great!

**Review by nickcabuno — APPROVED** [review]
awesome job fellas, everything looks great

**Review by devin-mitchell — APPROVED** [review]
couple  unnecessary comments we can clean up.  But due to time i'm merging this so UX/UI team can start styling.  I'll start working on a new PR for clean up.

**Pull request merged**